introduction_to_amazon_algorithms/jumpstart_image_classification/Amazon_JumpStart_Image_Classification.ipynb

```diff
@@ -0,0 +1,661 @@
+{
```

Comment:
<p>Why do we have <code>ImageNetLabels</code> at the end of the first bullet point? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
It provides hyper-link to the list of the ImageNetLabels.

Reply:
Ok. I would suggest using complete sentence instead of just saying the dataset name.

Reply:
I think it is fine, the previous sentence explains what is ImageNet.